Shared/Samples/Create and edit geometries/CreateAndEditGeometriesView.swift

```diff
@@ -274,39 +284,20 @@ class GeometryEditorMenuModel: ObservableObject {
         shouldUniformScale ? .uniform : .stretch
     }
     
-    /// Creates the geometry menu with a geometry editor.
-    /// - Parameter geometryEditor: The geometry editor that the menu should interact with.
-    /// - Parameter graphicsOverlay: The graphics overlay that is used to save geometries to.
-    init(geometryEditor: GeometryEditor, graphicsOverlay: GraphicsOverlay) {
-        self.geometryEditor = geometryEditor
-        self.graphicsOverlay = graphicsOverlay
-        
-        Task { [weak self, geometryEditor] in
-            for await geometry in geometryEditor.$geometry {
-                self?.geometry = geometry
-            }
-        }
-        Task { [weak self, geometryEditor] in
-            for await selection in geometryEditor.$selectedElement {
-                self?.selection = selection
-            }
-        }
-    }
-    
     /// Saves the current geometry to the graphics overlay and stops editing.
-    /// - Precondition: `canSave`
+    /// - Precondition: Whether or not the geometry is from a valid sketch.
```

Comment:
This precondition description doesn't make sense. It should describe what the condition is, ie. what would cause a failure.